src/app/globals.css

```
@@ -57,3 +57,14 @@
     @apply bg-background text-foreground;
   }
 }
+
+/* Remove focus ring from dropdowns */
+.select-trigger:focus,
+.select-trigger:focus-visible,
+[role="combobox"]:focus-visible,
+[data-state="open"]:focus-visible {
+  outline: none !important;
+  box-shadow: none !important;
+  ring-width: 0 !important;
```

Biome (1.9.4)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)